autoPyTorch/datasets/base_dataset.py

```diff
@@ -322,7 +322,7 @@ def create_holdout_val_split(
             self.random_state, val_share, self._get_indices(), **kwargs)
         return train, val
 
-    def get_dataset_for_training(self, split_id: int, train: bool) -> Dataset:
+    def get_dataset_for_training(self, split_id: int, train: bool, subset: int = 0) -> Dataset:
```

Comment:
just a suggestion, but do you think we can just use the train flag to use the subset. because if we have train=True we need 0 and if train=False we need 1?

Reply:
I agree with this suggestion.